docker-compose.yml

```diff
@@ -101,7 +101,7 @@ mapfishapp:
   image: georchestra/mapfishapp:latest
   volumes:
     - /etc/georchestra:/etc/georchestra
-    - mapfishapp_uploads:/var/local/uploads
+    - mapfishapp_uploads:/mnt/mapfishapp_uploads
```

Comment:
Ok so we need to update https://github.com/georchestra/datadir/blob/docker-master/mapfishapp/mapfishapp.properties#L8 with the new path : `/mnt/mapfishapp_uploads`